packages/js/components/src/collapsible-content/collapsible-content.tsx

```diff
@@ -33,27 +34,46 @@ export const CollapsibleContent: React.FC< CollapsedProps > = ( {
 		return persistRender ? 'visually-hidden' : 'hidden';
 	};
 
+	const collapsibleToggleId = useInstanceId(
+		CollapsibleContent,
+		'collapsible_toggle'
+	) as string;
+	const collapsibleContentId = useInstanceId(
+		CollapsibleContent,
+		'collapsible_content'
+	) as string;
+
+	const displayState = getState();
+
 	return (
-		<div
-			aria-expanded={ collapsed ? 'false' : 'true' }
-			className={ `woocommerce-collapsible-content` }
-		>
+		<div className="woocommerce-collapsible-content">
 			<button
+				type="button"
+				id={ collapsibleToggleId }
 				className="woocommerce-collapsible-content__toggle"
 				onClick={ () => setCollapsed( ! collapsed ) }
+				aria-expanded={ collapsed ? 'false' : 'true' }
+				aria-controls={
+					displayState !== 'hidden' ? collapsibleContentId : undefined
```

Comment:
Nice.  This makes me wonder if we should only use be using the `visually-hidden` and not provide both options since it's a simpler implementation and provides better support all the way around.

Not something that needs to be addressed in this PR.  Also `aria-controls` is now highly utilized by screen readers as far as I'm aware.